<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtHelpConfigUI</class>
 <widget class="QWidget" name="QtHelpConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>404</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="qchManage">
     <property name="title">
      <string>Manage QtHelp Documentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="qchTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="showGrid" stdset="0">
             <bool>false</bool>
            </property>
            <attribute name="headerVisible">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Path</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">IconName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">Ghns</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="qchSearchDir">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Search qch files in directory...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="KPushButton" name="addButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="editButton">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="upButton">
            <property name="text">
             <string>Move Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="downButton">
            <property name="text">
             <string>Move Down</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="qchSearchDirButton">
            <property name="text">
             <string>Choose dir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="loadQtDocsCheckBox">
        <property name="text">
         <string>Load Qt documentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
